<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EdgeExtractDlg</class>
 <widget class="QDialog" name="EdgeExtractDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="subPixelEnableCheckBox">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>573</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>亚像素使能</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxScale">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>133</y>
     <width>341</width>
     <height>130</height>
    </rect>
   </property>
   <property name="title">
    <string>算子选择</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <widget class="QRadioButton" name="sobelRadioBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>sobel</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="prewittRadioBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>prewitt</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="laplacianRadioBtn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>laplacian</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="robertsRadioBtn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>roberts</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxLimit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>383</y>
     <width>340</width>
     <height>180</height>
    </rect>
   </property>
   <property name="title">
    <string>门限值</string>
   </property>
   <widget class="QLabel" name="lowLimitValueLabel">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>100</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lowLimitDecBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="lowLimitSlider">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>135</y>
      <width>220</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="lowLimitIncBtn">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>120</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lowLimitLabel">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>100</y>
      <width>70</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>下门限：</string>
    </property>
   </widget>
   <widget class="QLabel" name="highLimitValueLabel">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>30</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="highLimitIncBtn">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>50</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="highLimitDecBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="highLimitLabel">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>30</y>
      <width>70</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>上门限：</string>
    </property>
   </widget>
   <widget class="QSlider" name="highLimitSilder">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>60</y>
      <width>220</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxDirection">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>263</y>
     <width>341</width>
     <height>120</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>边缘提取方向</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <widget class="QRadioButton" name="xRadioBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>X方向</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="yRadioBtn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Y方向</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="xyRadioBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>XY方向</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>720</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>完成</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnQuit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>720</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(108, 108, 108);</string>
   </property>
   <property name="text">
    <string>边沿提取</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxMode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>341</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>提取模式</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <widget class="QRadioButton" name="RadioBtnPoint">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>边沿点</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RadioBtnPic">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>边沿图</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
